### PR TITLE
ACP2E-954: Fixed the invalid filter definition for FilterMatchTypeInput

### PR DESCRIPTION
**Pull request by kavingas**
## Purpose of this pull request

This pull request (PR) ACP2E-954: Fixed the invalid filter definition for FilterMatchTypeInput

## Affected DevDocs pages
- https://devdocs.magento.com/guides/v2.3/graphql/custom-filters.html

### CODE REVIEW

File: src/guides/v2.3/graphql/custom-filters.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ You have several options when enabling a custom attribute (or any attribute that
 The [`filter`]({{page.baseurl}}/graphql/queries/products.html#ProductFilterInput) definition for your custom attribute requires one of the following input data types:
 
 -  `FilterEqualTypeInput` - Specify this data type when the **Catalog Input Type for Store Owner** field for your custom attribute is set to Yes/No, Select, or Multiple select. Your filter can contain the `eq` or `in` attribute. Use the `eq` attribute to exactly match the specified string. Use the `in` attribute to filter on a comma-separated list of values.
--  `FilterMatchTypeInput` - Specify this data type when the **Catalog Input Type for Store Owner** field for your custom attribute is set to Text Field or Text Area. Your filter must contain the `match` attribute, which will return all items that exactly match the specified string.
+-  `FilterMatchTypeInput` - Specify this data type when the **Catalog Input Type for Store Owner** field for your custom attribute is set to Text Field or Text Area. Your filter must contain the `match` attribute, which will defines a filter that performs a fuzzy search.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/custom-filters.md:21]:
```suggestion
-  `FilterMatchTypeInput` - Specify this data type when the **Catalog Input Type for Store Owner** field for your custom attribute is set to Text Field or Text Area. Your filter must contain the `match` attribute, which will define a filter that performs a fuzzy search.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Please also sign the Adobe CLA

**Comment by kavingas** [comment]
> Please also sign the Adobe CLA

I just signed the Adobe CLA

**Comment by keharper** [comment]
@kavingas This appears to be a duplicate of https://github.com/magento-commerce/devdocs/pull/3064. We'll reopen it if I'm wrong.

**Pull request closed without merging**